addons/mail/static/src/discuss/web/avatar_card/avatar_card_popover.xml

```diff
@@ -14,7 +14,33 @@
                         </span>
                     </span>
                     <div class="d-flex flex-column o_card_user_infos overflow-hidden">
-                        <span class="o-mail-avatar-card-name fw-bold" t-if="name" t-esc="name"/>
+                        <div class="o-mail-avatar-card-name-actions d-flex align-items-center justify-content-center">
+                            <span class="o-mail-avatar-card-name fw-bold" t-if="name" t-esc="name"/>
+                            <div class="flex-grow-1"/>
+                            <Dropdown t-if="props.channelMember?.channel_id?.channel.self_member_id?.channel_role == 'owner' or props.channelMember?.channel_id?.channel.self_member_id?.channel_role == 'admin'" position="'right-start'">
+                                <button class="btn btn-sm">
+                                    <i class="oi oi-ellipsis-v"/>
+                                </button>
+                                <t t-set-slot="content">
+                                    <t t-if="props.channelMember?.channel_id?.channel.self_member_id?.channel_role == 'owner'">
+                                        <t t-if="props.channelMember.channel_role !== 'owner'">
+                                            <DropdownItem t-if="props.channelMember.channel_role !== 'admin'" onSelected.bind="() => this.setChannelRole('admin')">
+                                                Set as Admin
+                                            </DropdownItem>
+                                            <DropdownItem t-if="props.channelMember.channel_role === 'admin'" onSelected.bind="() => this.setChannelRole(false)">
+                                                Remove Admin
+                                            </DropdownItem>
+                                            <DropdownItem onSelected.bind="() => this.setChannelRole('owner')">
+                                                Set as Owner
+                                            </DropdownItem>
+                                        </t>
+                                    </t>
+                                    <DropdownItem onSelected.bind="onClickRemove">
+                                        <span class="text-danger">Remove</span>
```

Comment:
I find the "Remove" text slightly confusing, especially in the avatar card out of context.

Maybe "remove form channel" or something more explicit.